### ISSUE TITLE
Hot reload for prefabs

### ISSUE THREAD

**Issue by rewin123** [rewin123/space_editor#99]
When changing a prefab file, it should reload in the editor/game window if it is opened via the PrefabLoader component

**Comment by naomijub** [comment]
bevy asset file watch might be a quick win here

**Comment by rewin123** [comment]
Yes. But you have to take the time to correctly re-spawn a prefab in a scene, especially if he or his children are referenced by other prefabs
